src/shared/services/dom-util.service.ts

```diff
@@ -32,6 +32,35 @@ export class DomUtilService {
       if (firstInput) {
         firstInput.focus();
         firstInput.select();
+      } else {
+        firstInput = el.querySelector('div.btn-group') as HTMLInputElement;
+        if (firstInput) {
+          // .open is used for opening the dropdownmenu
+          firstInput.classList.add('open');
+          firstInput = el.querySelector('div.btn-group input') as HTMLInputElement;
+          firstInput.focus();
+          firstInput.select();
+        }
+      }
+    }
+  }
+
+  // TO-DO: find the next editable element in row.
+  focusRightOrLeftCell(index: number, direction: number) {
+    let firstInput = document.querySelector('[tabIndex=\'' + index + '\']') as HTMLElement;
```

Comment:
@harunurhan after changed this it gets more complicated because using tabindex i could directly identify the input or textarea field that is actually editable. Otherwise with the path i get the div parent and it needs to check more stuff in DOM to check if the next/previous element is tabable. This change actually leads to @jmartinm comment about the ctrl + right/ctrl + left bug. Should i use the previous implementation?